code/modules/projectiles/projectile/special.dm

```diff
@@ -354,4 +354,36 @@ obj/item/projectile/kinetic/New()
 		return ..()
 	if(!gun)
 		qdel(src)
-	gun.create_portal(src)
\ No newline at end of file
+	gun.create_portal(src)
+
+/obj/item/projectile/snowball
+	name = "snowball"
+	icon_state = "snowball"
+	hitsound = 'sound/items/dodgeball.ogg'
+	damage = 3
+	damage_type = BURN
+
+/obj/item/projectile/snowball/on_hit(atom/target)	//chilling
+	if(istype(target, /mob/living))
+		var/mob/living/M = target
+		M.bodytemperature = max(0, M.bodytemperature - 50)	//each hit will drop your body temp, so don't get surrounded!

```

Comment:
Yeah, if there's a lot of snowmen, this will drive you to starvation very quickly, followed by incredibly slow speed and death to burn damage.